src/core/Akka.Persistence/Eventsourced.cs

```diff
@@ -695,6 +695,12 @@ public void Prepend(IEnumerable<Envelope> envelopes)
             {
                 _userStash.Prepend(envelopes);
             }
+
+            public int Count => _userStash.Count;
```

Comment:
For the `Eventsourced` stash, we use the values provided for the user stash rather than the internal one (since the user can't really affect the internal stash anyway.)